### PR TITLE
1.6.0 m6 missing bits

### LINKED ISSUE #94: Rollback workaround when releasing Lagom 1.6.0-M6
https://github.com/lagom/lagom-samples/pull/93 introduces a workaround for https://github.com/lagom/lagom/issues/2192 in the couchbase-scala-example. The correct fix should be part of Lagom `1.6.0-M6` so when bumping to that version we must rollback https://github.com/lagom/lagom-samples/pull/93/commits/a30882f1fb898962e673b431041cac7885215038

### PR DESCRIPTION
**Pull request by ignasi35**
related to #98  

Fixes #94

### CODE REVIEW

File: grpc-example/grpc-example-java/project/plugins.sbt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // The Lagom plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.6.0-M5")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.6.0-M6")
```
Comment (ignasi35) [inline grpc-example/grpc-example-java/project/plugins.sbt]: Not sure about this one. @dwijnand did you leave this particular sample out for incompatibilities with `play-grpc`?

Comment (dwijnand) [reply grpc-example/grpc-example-java/project/plugins.sbt]: Yes.  I suspect it'll need play-grpc to be updated, at some point.

Comment (ignasi35) [reply grpc-example/grpc-example-java/project/plugins.sbt]: Meh, I see now that the error is on the grpc generated code using the deprecated `ActorMaterializer` and this build being tuned to fail on `warn`. It'd be good to have this fixed in `play-grpc`.

Comment (ignasi35) [reply grpc-example/grpc-example-java/project/plugins.sbt]: I've removed this change from the PR and only left the fix for #94

Comment (ignasi35) [reply grpc-example/grpc-example-java/project/plugins.sbt]: Correction: the generated code introducing `ActorMaterializer` is on akka/akka-grpc.

**Review by dwijnand — APPROVED** [review]

**Pull request merged**